ci/program-anchor-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ci/rust-version.sh stable
+source ci/solana-version.sh install
+source ci/anchor-cliversion.sh install
+
+set -x
+
+cd account-compression && anchor build
```

Comment:
Since the name of this file is general, how about making behave more like `ci/cargo-test-sbf.sh`, but just where it takes a directory to `cd` into before running `anchor build`, ie:

```
usage() {
  exitcode=0
  if [[ -n "$1" ]]; then
    exitcode=1
    echo "Error: $*"
  fi
  echo "Usage: $0 [program-directory]"
  exit $exitcode
}

program_directory=$1
if [[ -z $program_directory ]]; then
  usage "No program directory provided"
fi

cd $program_directory
anchor build
```

So then in ci you do `./ci/anchor-build.sh account-compression`

Reply:
Added, thanks!